[13.0][IMP] sale_exception: Add customer sale warning + product warning exceptions (#1692)
OCA/sale-workflow: sale_exception/readme/USAGE.rst

Product warning:
#. Go to *Sales > Products > Products*.
#. Edit or create new product.
#. Go to *Sales* tab.

> #. Set warning option according to *Warning when Selling this Product* group.

Reviewer: ```suggestion
#. Set your desired warning option under the *Warning when Selling this Product* group.
```

#. Set some warning message.
#. Go to *Sales > Orders > Quotations*
#. Create new quotation.
#. Add product with warning message.
#. Confirm order.

> #. An exception will be displayed.

Reviewer: From my testing, the exception is displayed when saving the quotation, not when confirming the order, so maybe change the order here to make it clearer.
